Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -897,11 +897,10 @@ void GameListCtrl::OnRightClick(wxMouseEvent& event)
         auto* const uninstall_wad_item =
             popupMenu.Append(IDM_LIST_UNINSTALL_WAD, _("Uninstall from the NAND"));
         // These should not be allowed while emulation is running for safety reasons.
-        for (auto* menu_item : {install_wad_item, uninstall_wad_item})
-          menu_item->Enable(!Core::IsRunning() || !SConfig::GetInstance().bWii);
-
-        if (!WiiUtils::IsTitleInstalled(selected_iso->GetTitleID()))
-          uninstall_wad_item->Enable(false);
+        const bool can_enable = !Core::IsRunning() || !SConfig::GetInstance().bWii;
+        install_wad_item->Enable(can_enable);
+        uninstall_wad_item->Enable(can_enable &&
+                                   WiiUtils::IsTitleInstalled(selected_iso->GetTitleID()));
```

Comment:
That line break...any thoughts on `const bool is_installed = WiiUtils::IsTitleInstalled(selected_iso->GetTitleID())` and `uninstall_wad_item->Enable(can_enable && is_installed)`? I can't imagine this being too performance critical to not just call that method unconditionally.